src/services/completions.ts

```diff
@@ -472,6 +474,15 @@ namespace ts.Completions {
             return undefined;
         }
 
+        if (originIsExport(origin)) {
+            data = {

```

Comment:
If this is just used as a disambiguator, I think there's a better name than CompletionEntry `-Data`, like CompletionEntry -key, --handle, -identifier, -reference, -lookup.

Later: After reading the rest of the changes, I still think there's a better name somewhere, but I don't know what it is.

Reply:
It’s named to match the LSP spec.

Reply:
I guess the type name could be more indicative of what we’re using it for, but the property name should remain `data`.